examples/metrics/kpi/README.md

```diff
@@ -0,0 +1,122 @@
+# Helidon Metrics Key Performance Indicators SE Example
+
+This project implements a simple Hello World REST service using Helidon SE and demonstrates 
+support in Helidon for extended key performance indicator (KPI) metrics.
+
+Your application can set up KPI metrics either programmatically or using configuration.
+The `Main` class of this example shows both techniques, checking the system property `useConfig` to 
+determine 
+which to use.
+You would typically write any given application to use only one of the approaches.
+
+## Build and run
```

Comment:
[not a specific comment for this readme, just an observation]
Title case is typically used for H1 and H2 while sentence case is reserved for H3/4. I see that this is done differently all over the site and repo, but I think we could implement some standards going forward. Need to speak to Joe, Romain and Dmitry.